DOC-1652 change from VPC peering to PSC (#408)
redpanda-data/cloud-docs: modules/networking/pages/dedicated/gcp/configure-psc-in-api.adoc

== Enable Private Service Connect on an existing cluster

[CAUTION]
====

> As soon as Private Service Connect is available on your VPC, all communication on existing Redpanda bootstrap server and  broker ports is interrupted due to the change on the private DNS resolution. Make sure all applications running in your VPC are ready to start using the corresponding Private Service Connect ports. 

Reviewer: Just a suggestion.

```suggestion
Enabling Private Service Connect on your VPC interrupts all communication on existing Redpanda bootstrap server and broker ports due to the change of private DNS resolution.
```

Author: thanks, fixed!
